cmd/podman/kube/down.go

```diff
@@ -0,0 +1,61 @@
+package pods
+
+import (
+	"io"
+	"io/ioutil"
+	"os"
+
+	"github.com/containers/podman/v4/cmd/podman/common"
+	"github.com/containers/podman/v4/cmd/podman/registry"
+	"github.com/spf13/cobra"
+)
+
+var (
+	downDescription = `Reads in a structured file of Kubernetes YAML.
+
+  Removes pods that have been based on the Kubernetes kind described in the YAML.`
+
```

Comment:
I'm guessing we don't have enough room here to list the valid kinds as the man pages do?

Reply:
We need to do improve the docs for all-things-kube but I think it's best to do that in a separate PR.